pack/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+cd "${0%/*}/.."
+
+TAG="${VERSION:-latest}"
+IMAGE="${IMAGE:-cdf-tk}"
+PUBLISH="${PUBLISH:-false}"
+
+echo "Building image $IMAGE:$TAG"
+
+set +e
+if $PUBLISH -eq "true"; then
+  IMAGE="cognite/cdf-tk"  # official name of the tool within Docker Hub
```

Comment:
Not sure what common practice is, but maybe name this similar to the python package `cognite-toolkit` 
```suggestion
  IMAGE="cognite/toolkit"  # official name of the tool within Docker Hub
```

Reply:
@gregertw Any opinions about this one? I think this is the most important part of the PR as we should not change the name of the image after it has been published.

Reply:
Agree, toolkit is the most appropriate since cdf-tk is only the tool, but we also pack in the templates: so the entire toolkit.